src/theme.ts

```diff
@@ -46,10 +46,13 @@ export const theme = {
   },
   borderRadii: {
     md: "7px",
-    md_small: "4px",
-    b: "none",
+    sm: "4px",
+    border: "none",
+    seam: "4px solid rgba(0, 0, 0, 0.1)",
+  },
+  boxShadow: {
     bs: "rgba(0, 0, 0, 0.15) 2.4px 2.4px 3.2px",
```

Comment:
Now that there is a `thick` maybe this can be `light` or `medium`? I don't like `boxShadow.bs`, where `bs` is just short for box-shadow.